src/containers/offer-details/offer-carousel/OfferCarousel.tsx

```diff
@@ -70,9 +70,16 @@ const OfferCarousel: FC<OfferCarouselProps> = ({ offer }) => {
         }
       }
     ]
+    const onBookmarkClick = (id: string) => {
+      console.log(id)
```

Comment:
the same as we discussed on meeting(on my PR), try to replace it by returning "null"